src/components/NftRewards/NftRewardsSection.tsx

```diff
@@ -72,11 +72,27 @@ export function NftRewardsSection() {
   const payAmountETH =
     payInCurrency === ETH ? payAmount : fromWad(converter.usdToWei(payAmount))
 
-  const handleTierDeselect = (tierId: number | undefined) => {
+  const handleTierDeselect = (
+    tierId: number | undefined,
+    quantity: number | undefined, // quantity to deselect. Remove all instances of tierId is undefined
```

Comment:
**suggestion** what about we enforce quantity to be defined, and call `handleTierDeselect` with quantity=0 when we want to deselect it completely